### PR TITLE
Slave unit removed in modbus service calls.

### PR DESCRIPTION
**Pull request by janiversen**
## Proposed change

Remove duplicate parameter "unit" (duplicate of "slave").

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/99285
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -1538,7 +1538,6 @@ Description:
...
 | hub       | Hub name (defaults to 'modbus_hub' when omitted)                                                                                                                                                                                                                                            |
-| unit      | Slave address (0-255), alternative to slave                                                                                                                                                                                                                                                 |
 | slave     | Slave address (0-255), alternative to unit                                                                                                                                                                                                                                                  |
```
Comment (emontnemery) [inline source/_integrations/modbus.markdown]:
```suggestion
| slave     | Slave address (0-255)                                                                                                                                                                                                                                                  |
```

**Review by emontnemery — CHANGES_REQUESTED** [review]
Please also update the "writing a float32 type register" example

**Comment by janiversen** [comment]
@emontnemery done.

**Review by emontnemery — APPROVED** [review]
Thanks, @janiversen 👍

**Pull request merged**